src/Core/Services/Implementations/UserService.cs

```diff
@@ -1263,21 +1263,29 @@ public async Task<bool> IsLegacyUser(string userId)
         return IsLegacyUser(user);
     }
 
-    public async Task<bool> IsManagedByAnyOrganizationAsync(Guid userId)
+#nullable enable
+    public async Task<bool?> IsManagedByAnyOrganizationAsync(Guid userId)
     {
-        var managingOrganization = await GetOrganizationManagingUserAsync(userId);
-        return managingOrganization != null;
+        var managingOrganizations = await GetOrganizationsManagingUserAsync(userId);
+        return managingOrganizations?.Any();
     }
 
-    public async Task<Organization> GetOrganizationManagingUserAsync(Guid userId)
+    public async Task<IEnumerable<Organization>?> GetOrganizationsManagingUserAsync(Guid userId)
```

Comment:
I suggest that this is also made non-nullable. You can just return an empty list if the feature flag is off.

Reply:
I changed it to return an empty list, cleanup will be easier later on.